### ISSUE TITLE
Add label to auto-size for screen readers

### ISSUE THREAD

**Issue by seejwy** [broadwayinc/skateui#75]
Autosize does not have a label which is difficult for people who are visually handicapped to understand the context of the content through screen readers. Place a visibility hidden label on autosize to be read by screen readers.

**Comment by broadwayinc** [comment]
label is a separate element rather than its props.
why not element id to be inherited to textarea element so the enduser can create separate label element to point to the textarea id?

**Comment by baksagimm** [comment]
resolved in #91
you can create label element to point to autosize component.
